### DISCUSSION TITLE
Practica HTML (Sugerencias)

### DISCUSSION THREAD

**Discussion by KevinRomero2** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#263] [General]
Actualmente, he terminado la práctica de HTML y pude desplegar y aplicar un poco De CSS al sitio a lo que voy aprendiendo soy nuevo en esto, y ahí voy poco a poco, así que dejaré el link del sitio para quienes gusten verlo. Y como dicen no desanimase si ven sitios más profesionales una va a su propio ritmo.

Este es el [LINK](https://cuycakretions.000webhostapp.com/index.html)

Se aceptan comentarios, sugerencias y Retroalimentación, ya que si ven que puede mejorar algo o aplicar más CSS, si me lo hacen saber en los comentarios serian genial, porque eso me permitiría aprender más y sumar a mi aprendizaje, se los agradecería mucho,

**Comment by RocioGH** [comment]
Hola, te quedó muy padre, gracias por mostrar tu práctica, la verdad  yo también soy nueva voy empezando desde cero y espero poder hacer mi práctica no tan fea jejejejejee

**Reply by KevinRomero2** [reply]
Animo Muchas gracias, y claro que se puede :D

**Reply by MARYCHS** [reply]
Te quedo muy padre, gracias por compartirnos tu creatividad.

**Reply by KevinRomero2** [reply]
Muchas gracias @MARYCHS

**Comment by osorniochris** [comment]
Muchas felicidades compañero, tu trabajo cumple las especificaciones que se solicitaron.

Mi recomendación para mejorar las habilidades de css es 'tomar inspiración de los profesionales', es un buen ejercicio elegir un sitio web existente y clonar el diseño (como la misión css). 

Te dejo algunos recursos útiles que me han servido.

*En [Dribbble](https://dribbble.com/) también puedes encontrar una gran variedad de ideas y diseños, el chiste es identificar los componentes e investigar como puedes hacerlos tú mismo :)
*Este [sitio](https://jgthms.com/web-design-in-4-minutes/) te enseña a mejorar el diseño de una página en 4 minutos jaja
*Twitter es uno de los mejores sitios para seguir desarrolladores y diseñadores, te dejo al buen [Steve Schoger](https://twitter.com/i/events/994601867987619840)
*Si eres más audiovisual, este compa es de los mejores que yo he visto [Design Course](https://www.youtube.com/designcourse)
*Para finalizar este jueguito que trata de evaluar cual de las UI tiene un mejor diseño para el usuario, está curioso [Can't Unsee](https://cantunsee.space/)

**Reply by KevinRomero2** [reply]
wow muchísimas gracias que buena información, links muy valiosos e importantes para el aprendizaje, gracias por compartir ese conocimiento.

**Comment by DanyVeneno** [comment]
Que tal Kevin muy bien cada quien a su ritmo la meta es que le entiendas y vayas puliendo y agregarle tus propios retos si necesitas algo y  se como se hace no lo dudes aqui andamos saludos buena tarde.

**Comment by JacomeGerardo** [comment]
En donde dice "maquetación de páginas del pastelero", eso es otra página web aparte? o cómo? Gracias por su respuesta desde ya!

**Reply by Ingrzs** [reply]
Maquetación solo es poner los elementos html, sin color sin nada y si es el sitio web pastelero que tiene 4 paginas o dependerá de ti cuantas les pones, porque debes tener la pagina que usarán los clientes y la que usara el administrador.

**Comment by hrrRafael2** [comment]
Hola les quiero mostrar mi web , no soy muy bueno para el diseño  , pero ya quedo... 
Acepto recomendaciones , para seguir mejorando. [Link](https://hrrrafael2.github.io/html-practica-pasteleria/)

**Comment by MARYCHS** [comment]
Esta muy padre.  Compraba todos.  Solo hay unos pasteles que valen 200 y
otros 0 pesos y en contacto como que esta en ruso o no se entiende.

Excelente, a mi no me quedo tan bonito.

**Comment by OscarPedroza** [comment]
Bastante bien el sitio, muy buena temática también, arriba los cuyos jeje.

**Reply by KevinRomero2** [reply]
Ojo es que los cuyos Son geniales
